### PR TITLE
Remove applicationCluster from tables.ts

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Should hopefully be the last ones

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: components/ws-manager-bridge/src/workspace-instance-controller.ts [hunk 1]
```diff
@@ -144,7 +137,7 @@ export class WorkspaceInstanceControllerImpl implements WorkspaceInstanceControl
         const span = TraceContext.startSpan("controlNonStoppedWSManagerManagedInstances", parentCtx);
         const ctx = { span };
         try {
-            log.debug("Controlling ws-manager managed instances...", { workspaceClusterName });
+            log.debug("Controlling ws-manager managed instances...");
```
Comment (easyCZ) [inline components/ws-manager-bridge/src/workspace-instance-controller.ts:140]: The clusterName was only used in logs

Comment (geropl) [reply components/ws-manager-bridge/src/workspace-instance-controller.ts:140]: But that's also very important part of the logs :laughing: I used it multiple times in the past to rule out errors cases.

Why remove it in the first place? :thinking: Also, it's independent of the change, right?

Comment (easyCZ) [reply components/ws-manager-bridge/src/workspace-instance-controller.ts:140]: Technically, the value that's supplied is the value of https://github.com/gitpod-io/gitpod/pull/16876/files#diff-e7027a2857b316687949201e4fbd013e2bea4d59b0cce596c4f6fa997bdff2e3L51

Which is always the current application cluster. The naming in this method is misleading, as it's not the workspaceClusterName, but it's actually the application cluster. And given we only have one, it's always the same value.

File: components/ws-manager-bridge/src/workspace-instance-controller.ts [hunk 2]
```diff
@@ -204,22 +197,19 @@ export class WorkspaceInstanceControllerImpl implements WorkspaceInstanceControl
     async controlNotStoppedAppClusterManagedInstanceTimeouts(
         parentCtx: TraceContext,
         runningInstances: RunningWorkspaceInfo[],
-        applicationClusterName: string,
     ) {
         const span = TraceContext.startSpan("controlNotStoppedAppClusterManagedInstanceTimeouts", parentCtx);
         const ctx = { span };
         try {
-            log.debug("Controlling app cluster managed instances...", { applicationClusterName });
```
Comment (geropl) [inline components/ws-manager-bridge/src/workspace-instance-controller.ts:212]: This distinction is actually important. I see no reason to remove it.

Comment (easyCZ) [reply components/ws-manager-bridge/src/workspace-instance-controller.ts:212]: But the value it contains is always going to be `us-02`, no?

File: components/ws-manager-bridge/src/workspace-instance-controller.ts (hunk 1 above)
Comment (geropl) [reply components/ws-manager-bridge/src/workspace-instance-controller.ts:140]: I think somebody of us is confusing something: I see this method being called [here](https://github.com/gitpod-io/gitpod/blob/495dab06d756610a60256f732badf352490476b5/components/ws-manager-bridge/src/workspace-instance-controller.ts#L91-L97), using the workspaceClusterName from [here](https://github.com/gitpod-io/gitpod/blob/495dab06d756610a60256f732badf352490476b5/components/ws-manager-bridge/src/workspace-instance-controller.ts#L72)? :thinkin:

File: components/ws-manager-bridge/src/workspace-instance-controller.ts (hunk 2 above)
Comment (geropl) [reply components/ws-manager-bridge/src/workspace-instance-controller.ts:212]: I mean the "Controlling **app cluster** managed instances..."

Comment (geropl) [reply components/ws-manager-bridge/src/workspace-instance-controller.ts:212]: I agree on the applicationClusterName, that we can ditch. :+1:

File: components/ws-manager-bridge/src/workspace-instance-controller.ts (hunk 1 above)
Comment (geropl) [reply components/ws-manager-bridge/src/workspace-instance-controller.ts:140]: :dart:

**Review by geropl — APPROVED** [review]

**Pull request merged**